src/router/storeSync/CrossHairParamConfig.class.js

```diff
@@ -22,6 +23,17 @@ function dispatchCrossHairFromUrlIntoStore(to, store, urlParamValue) {
                     dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN,
                 })
             )
+        } else if (parts.length === 2) {
+            const crossHairPosition = [parseFloat(parts[0]), parseFloat(parts[1])]
+            if (!(isNaN(crossHairPosition[0]) || isNaN(crossHairPosition[1]))) {
+                promisesForAllDispatch.push(
+                    store.dispatch('setCrossHair', {
+                        crossHair: CrossHairs.marker,
+                        crossHairPosition,
+                        dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN,
+                    })
+                )
+            }
```

Comment:
Where is the spec for the crosshair parameter ? If I understand correctly your code it is like this ?

- crosshair=MARKER,X,Y
- crosshair=X,Y

Was it always like this ? I think this is not a good idea and to be consistent with other parameters like the layers, the second case should be `crosshair=,X,Y` to use the default marker.

Reply:
Is this code really needed for which use case ?

Reply:
The crosshair parameters used to have two ways of being handled : 
`crosshair=MARKER` would add the marker in the center
`crosshair=MARKER,X,Y` would add the marker in a specific position

During the tests, Pascal tried to only use the coordinates in the links, and we decided to add the `crosshair=X,Y` too, as thinking "I want the crosshair to be in such position" is quite instinctive.

I like your idea of a `crosshair=,X,Y` to be consistent with others similar parameters, but that would require adding the crosshair to the legacy parser. @pakb , what do you think ?

Reply:
yep, go for the `,X,Y`, that is more consistent with what we already do on the `layers` URL param